improver_tests/acceptance/SHA256SUMS

```diff
@@ -633,73 +633,73 @@ e653ae271dd789dde8b03e6237cc7b48552dd9d5206d3430ee68d283683247a0  ./wind_downsca
+c832f6fa079edfa5d3f2447e70e6f52e659ebec3020a1fd4446a3c24e29d1f43  ./wxcode-modal/gridded_input/20201209T1400Z-weather_symbols-PT01H.nc
+c682e214ad69ca6d839598700749c51154fa51cb966416176b75b7cb010c42e4  ./wxcode-modal/gridded_input/20201209T1500Z-weather_symbols-PT01H.nc
+ad9255bfc7b7d9cabfd94bf4b34c3672341550fb51908d7573ebbd05b18dbb94  ./wxcode-modal/gridded_input/20201209T1600Z-weather_symbols-PT01H.nc
+2b0e58061890bfd8c2cb7478709fa8f36e7fecfb152d861f40deacaf046c761e  ./wxcode-modal/gridded_input/20201209T1700Z-weather_symbols-PT01H.nc
+fa3790f3fbafbacd056431faff9e552157c74dd10d28cdf194cbc12b77747ff6  ./wxcode-modal/gridded_input/20201209T1800Z-weather_symbols-PT01H.nc
+9de93fe11cff75430149d9db782e6540807737914bf480819b826c1bbbf64831  ./wxcode-modal/gridded_input/kgo.nc
+887c4b86c787426854d73a7a59b5b3b02edfe58310c463cb31c5946ccb4b0708  ./wxcode-modal/gridded_ties/20201209T0700Z-weather_symbols-PT01H.nc
+689f7714e15cb61103b96fb5c2cf6acd2868e3079cc92092367ff8374b8cbe59  ./wxcode-modal/gridded_ties/20201209T0800Z-weather_symbols-PT01H.nc
+fe26ff7ff90a88850f7a591ae1d3b5fcf587eaddd097ec180d91609477bd3578  ./wxcode-modal/gridded_ties/20201209T0900Z-weather_symbols-PT01H.nc
+b0c414932ef347d8832a85e481470de5610f629e2f9be7dce2b242118635b6ce  ./wxcode-modal/gridded_ties/20201209T1000Z-weather_symbols-PT01H.nc
+add8b53135bbf3ed9f84ed072a7cdfbed37ef8732b015b3eff62928433aa9985  ./wxcode-modal/gridded_ties/20201209T1100Z-weather_symbols-PT01H.nc
+17d06f40c28ed0b72fdeafeb8a5ca62f291a7389a9f9cc626a43d5f64fffa5c8  ./wxcode-modal/gridded_ties/20201209T1200Z-weather_symbols-PT01H.nc
+90d32b86caec048ec3b463221a17ee98ce9864d0661d27e1f4bb456909a406b4  ./wxcode-modal/gridded_ties/20201209T1300Z-weather_symbols-PT01H.nc
+cc5ab7c519438b376cec8d8c94e4f85a4faac62c890dea27c0744760ef194ea5  ./wxcode-modal/gridded_ties/20201209T1400Z-weather_symbols-PT01H.nc
+59a6a1c2bd5acf8d4328d09e5505fe5625470ccde8bb94205dd415aded564162  ./wxcode-modal/gridded_ties/20201209T1500Z-weather_symbols-PT01H.nc
+bcd21dd36dff6e6e790d332395d47c84847709a957ae4f47fc02147d65f8d698  ./wxcode-modal/gridded_ties/20201209T1600Z-weather_symbols-PT01H.nc
+dcde6de571798f469e01cb6e419c0a8a108d550a753a199cfa8e34e7a109c4f0  ./wxcode-modal/gridded_ties/20201209T1700Z-weather_symbols-PT01H.nc
+74f20abd59726b8f066ee32ad94fea05f37e5fe895fef5a6a0d3db722ce17cf5  ./wxcode-modal/gridded_ties/20201209T1800Z-weather_symbols-PT01H.nc
+83eb19444bc11d8c05b5db43ea993419f6e11a04e7fffae422630b8e902d8cdf  ./wxcode-modal/gridded_ties/kgo.nc
+08e509a106716d14992ac44f0b5864c488c9f329c098bdaf8e4686dff99019cb  ./wxcode-modal/single_input/20201210T0000Z-weather_symbols-PT01H.nc
+d7497c9f1b6e9cb93df59c168fc1e38a75a35ae27b654279dddd72a6e77867f3  ./wxcode-modal/single_input/kgo.nc
+80c3824c422debfd717e99c076e9a53bffa368e1591dbd3a3bbf83649b015dc6  ./wxcode-modal/spot_input/20201209T0700Z-weather_symbols-PT01H.nc
```

Comment:
In addition to the `mosg__model_configuration` attribute which looks like it has been updated correctly, it looks like the `title` has also been updated. This updated `title` attribute doesn't seem consistent with the suite (unlike the `title` attribute in the master acceptance test files). Could this be corrected?

Reply:
I changed these to make the interpreter happy. It seems that what comes out of the suite currently is inconsistent with the interpreter. I've asked Bruce to clarify what we should have here.

Reply:
Bruce has clarified that the title attribute should be `'IMPROVER Post-Processed Multi-Model Blend UK Spot Values'` and therefore the suite is wrong. I have created a suite PR to fix the spot titles there and updated these titles to match.